src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -66,17 +111,20 @@ object AddPropertiesToDeposit {
       .getOrElse(Failure(ActionException(row, """The column "DEPOSITOR_ID" is not present""")))
   }
 
-  def writeProperties(row: Int, datasetID: DatasetID, dataset: Dataset)(implicit settings: Settings): Try[Unit] = {
-    val props = new Properties
+  def writeProperties(row: Int, datasetID: DatasetID, dataset: Dataset, emailaddress: String)(implicit settings: Settings): Try[Unit] = {
+    val props = new Properties {
+      // Make sure we get sorted output, which is better readable than random
+      override def keys(): util.Enumeration[AnyRef] = Collections.enumeration(new util.TreeSet[Object](super.keySet()))
```

Comment:
can't you just write these without `util`? If you explicitly want them to indicate that you are talking about a Java class, make it `ju.Enumeration` and `ju.TreeSet`, as is conventional. You do this by adding this line: `import java.{util => ju}`

I like that you create a sorted `Properties` output with this!